noto_license.txt

```diff
@@ -0,0 +1,48 @@
+Copyright (c) 2010-2014, Łukasz Dziedzic (dziedzic@typoland.com),
```

Comment:
Do we really need this in the root folder too, you seem to include this same license in every addon folder next to the actual used font.

Reply:
Not really, will remove here as on addons are enough :sweat_smile:.